sysdweb/server.py

```diff
@@ -32,6 +32,19 @@ def login(user, password):
     # Validate user with password
     return pam().authenticate(user, password)
 
+
+@hook('after_request')
+def enable_cors():
+    response.headers['Access-Control-Allow-Origin'] = '*'
+    response.headers['Access-Control-Allow-Methods'] = 'GET, OPTIONS'
+    response.headers['Access-Control-Allow-Headers'] = 'x-api-key, Content-Type, Authorization'
+
+
+@route('/api/v1/<_service>/<_action>', method='OPTIONS')
+def service_options_handler(_service, _action):
+    return {}
```

Comment:
To `OPTIONS` requests is better return a [204 No Content](https://apps.moire.org/httpstatus/204/).

You can change this to:
```python
@route('/api/v1/<_service>/<_action>', method='OPTIONS')
def service_options_handler(_service, _action):
    response.status = 204
```

Reply:
@ogarcia done
one potential modification could be to use a wild card url so all requests are affected, what do you think?
also it may make sense to add a config option to reject cors requests in the future

Reply:
Is a great idea to use a wildcard to enable CORS to all API calls. The config option, I think that is not necessary, I don't think there is a problem with CORS being always active.